test/e2e/storage/flexvolume_mounted_volume_resize.go

```diff
@@ -35,6 +36,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 )
 
+const (
+	// total time to wait for cloudprovider or file system resize to finish
+	totalResizeWaitPeriod = 20 * time.Minute
+)
```

Comment:
Lets reduce this time to like 5 minutes.

Reply:
Updated.